db/migrate/20210608161920_add_content_id_to_subscriber_lists.rb

```diff
@@ -0,0 +1,6 @@
+class AddContentIdToSubscriberLists < ActiveRecord::Migration[6.1]
+  def change
+    add_column :subscriber_lists, :content_id, :uuid
+    add_index :subscriber_lists, :content_id, unique: true
```

Comment:
Don't make this a `unique` index.  I can't think why we'd want two subscriber lists matching the same content ID, but I'm not confident enough in how email-alert-api works to say we can rule that out.